### DISCUSSION TITLE
How do I sort pages?

### DISCUSSION THREAD

**Discussion by BrandgrandRealMe** [squidfunk/mkdocs-material#5177] [Q&A] [answered]
Like if I have:
 - Years
    - 2010
    - 2014
    - 2020
 
 How do I sort it to be:
 - Years
    - 2020
    - 2014
    - 2010

**Comment by kamilkrzyskow** [comment, marked as answer]
By default there is no special way to do that. You have to structure your files and folders in a special way, and omit defining a `nav` in the `mkdocs.yml` it will create a sorted nav tree, or use the `awesome-pages` plugin for more control. If you're using some other plugin already that uses the structure provided above, maybe there is an option there.🤔 

Same answer as here (found by looking for `sort pages` 👁️): https://github.com/squidfunk/mkdocs-material/discussions/4684
Also you might read my answer here: https://github.com/squidfunk/mkdocs-material/discussions/5178
Plugin: https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin
